content/applications/hr/employees/offboarding.rst

```diff
@@ -0,0 +1,77 @@
+===========
+Offboarding
+===========
+
+When an employee leaves the company, it is important to ensure their employee record is updated to
+reflect their departure, log the reason why, close any open activities associated with the
+employee, and provide them with any important documents.
+
+Archive an employee
+===================
+
+In Odoo, when an employee leaves the company they must be *archived*. To archive an employee, first
+navigate to the main **Employees** app. From here, locate the employee who is leaving the company,
+and click on their employee card.
+
+The employee form loads, displaying all their information. Click the :icon:`fa-gear`
+:guilabel:`(gear)` icon in the top-left corner, and a drop-down menu appears. Click
+:icon:`oi-archive` :guilabel:`Archive`, and an :guilabel:`Employee Termination` pop-up window
+appears.
+
+Fill out the following fields on the form:
+
+- :guilabel:`Departure Reason`: Select a reason the employee is leaving from the drop-down menu. The
+  default options are:
+
+  - :guilabel:`Fired`: Select this option when an employee is being let go, and the company has
+    given notice.
+  - :guilabel:`Resigned`: Select this option when the employee no longer wishes to be employed, and
+    the employee has given notice.
+  - :guilabel:`Retired`: Select this option when the employee is retiring.
+  - :guilabel:`Became Freelance`: Select this option when the employee is no longer working for the
+    company, but is becoming a freelance worker instead.
+
+- :guilabel:`Contract End Date`: Using the calendar selector, select the last day the employee is
+  working for the company.
+- :guilabel:`Detailed Reason`: Enter a short description for the employee's departure.
+- :guilabel:`Close Activities`: Tick the checkbox next to each type of activity to close or delete
+  any open activities associated with it. It is recommended to tick **all** boxes that are
+  applicable. The available options are:
+
+  - :guilabel:`Appraisals`: Cancels all appraisals scheduled after the contract end date.
+  - :guilabel:`Contract`: Applies an end date for the current contract.
+  - :guilabel:`Company Car`: Removes the employee as the driver for their current company car, and
+    :ref:`assigns the next driver <fleet/new_vehicle/new-driver>`, if applicable.
+  - :guilabel:`Time Off`: Cancels any time off requests after the contract end date.
+  - :guilabel:`Allocations`: Removes the employee from any accrual plans they are on.
```

Comment:
```suggestion
  - :guilabel:`Appraisals`: cancels all appraisals scheduled after the contract end date.
  - :guilabel:`Contract`: applies an end date for the current contract.
  - :guilabel:`Company Car`: removes the employee as the driver for their current company car, and
    :ref:`assigns the next driver <fleet/new_vehicle/new-driver>`, if applicable.
  - :guilabel:`Time Off`: cancels any time off requests after the contract end date.
  - :guilabel:`Allocations`: removes the employee from any accrual plans they are on.
```
opting for lowercase in this instance, since the descriptions following the colon aren't complete sentences.